docs/4-Commands/kool-init.md

```diff
@@ -1,6 +1,6 @@
 ## kool init
 
-[DEPRECATED] Initialize kool preset in the current working directory. If no preset argument is specified you will be prompted to pick among the existing options.
```

Comment:
@danielsuguimoto is it the same case here? Should I revert this change?

Reply:
@leandroncbrito yep